src/test/java/uk/ac/ebi/eva/pipeline/ApplicationTest.java

```diff
@@ -43,14 +40,14 @@
     private JobExplorer jobExplorer;
 
     @Autowired
-    private JobOptions jobOptions;
+    private ParametersFromProperties parametersFromProperties;
```

Comment:
why not a DatabaseParameters? I don't like that method `ParametersFromProperties.getDatabaseName()`